src/encoded/commands/deploy.py

```diff
@@ -602,13 +579,15 @@ def hostname(value):
         help="ssh identity file path"
     )
     parser.add_argument('-b', '--branch', default=None, help="Git branch or tag")
+    parser.add_argument('--build-new-config', action='store_true', help="Build cloud config yaml.")
     parser.add_argument('-n', '--name', default=None, type=hostname, help="Instance name")
     parser.add_argument('--dry-run-aws', action='store_true', help="Abort before ec2 requests.")
+    parser.add_argument('--dry-run-cc', action='store_true', help="Abort after building cloud config.")
     parser.add_argument('--single-data-master', action='store_true',
             help="Create a single data master node.")
-    parser.add_argument('--check-price', action='store_true', help="Check price on spot instances")
     parser.add_argument('--cluster-name', default=None, help="Name of the cluster")
     parser.add_argument('--cluster-size', default=2, help="Elasticsearch cluster size")
+    parser.add_argument('--conf-dir', default='./cloud-config', help="Location of cloud build config")
     parser.add_argument('--elasticsearch', default=None, help="Launch an Elasticsearch instance")
```

Comment:
Do we want to change this to a boolean now? instead of a text string == "yes"?

Reply:
oh the elasticsearch line you mean, yea we can do that finally.